### ISSUE TITLE
I believe **token_embeddings_unnormalized** should be output_states from above?

### ISSUE THREAD

**Issue by seast** [Ginjing-Yuan/QWen2-from_ground_up#2]
I believe **token_embeddings_unnormalized** should be output_states from above?
```
second_normalized = rms_norm(token_embeddings_unnormalized, model["model.layers.0.post_attention_layernorm.weight"])
```
![Snip20240711_9](https://github.com/Ginjing-Yuan/QWen2-from_ground_up/assets/356810/01a6b3e1-36ab-432d-98ac-fc822c0862d7)

**Comment by Ginjing-Yuan** [comment]
Yes! Thank you very much for pointing out my mistake. I check the code and you are definitely right, I'm going to change it right now!!
